unstract/sdk1/src/unstract/sdk1/adapters/base1.py

```
@@ -639,6 +639,64 @@ def validate_model(adapter_metadata: dict[str, "Any"]) -> str:
+        has_thinking_config = (
+            "thinking" in adapter_metadata
+            and adapter_metadata.get("thinking") is not None
+        )
+        if not enable_thinking and has_thinking_config:
+            enable_thinking = True
+
+        if enable_thinking:
+            if has_thinking_config:
+                result_metadata["thinking"] = adapter_metadata["thinking"]
+            else:
+                thinking_config: dict[str, "Any"] = {"type": "enabled"}
```

Ruff (0.15.9)

[warning] Remove quotes from type annotation

Remove quotes

(UP037)